build.cmd

```diff
@@ -0,0 +1,4 @@
+cargo fmt -- --write-mode=overwrite
+cargo build
```

Comment:
Is there a way to write this so it's interoperable across os? I usually use bash (`.sh`) scripts. Also I generally like to put the scripts in their own directory, so `scripts/build.cmd` and `scripts/clean.cmd`.